lib/matplotlib/figure.py

```diff
@@ -1368,7 +1368,7 @@ def add_subplot(self, *args, **kwargs):
             if not args:
                 args = (1, 1, 1)
             # Normalize correct ijk values to (i, j, k) here so that
-            # add_subplot(111) == add_subplot(1, 1, 1).  Invalid values will
+            # add_subplot(211) == add_subplot(2, 1, 1).  Invalid values will
```

Comment:
Why is this now 2,1,1?

Reply:
I thought it would be clearer to use a value other than 1, 1, 1 since that is the special default above, but the comment and following code are for any value.